src/effects/VST3/VST3Effect.cpp

```diff
@@ -38,27 +38,21 @@
 
 #include "ShuttleGui.h"
 
-#include "memorystream.h"
 #include "VST3Utils.h"
 #include "VST3ParametersWindow.h"
 #include "VST3OptionsDialog.h"
+#include "VST3Wrapper.h"*
 
 #ifdef __WXMSW__
 #include <shlobj.h>
 #elif __WXGTK__
 #include "internal/x11/SocketWindow.h"
 #endif
 
-#include "AudacityVst3HostApplication.h"
 #include "ConfigInterface.h"
-#include "VST3ParametersWindow.h"
```

Comment:
I think it is the fifth commit that introduces the build failure on Linux.  It seems "Always" is a macro defined somewhere in one of the inclusions.

I was able to fix Linux builds of my cherry-picks of your branch by moving the definition of `VST3Effect::RealtimeSupport` earlier in the file, before other `#include` directives.  Another thing I didn't try was just to insert `#undef Always` before the function.

Reply:
Name collision caused by indirect include of X.h, fixed by moving includes to another file

Reply:
I see you added the build fix, but maybe it would be better done earlier in the commit sequence to keep all commits buildable.